common/scummsys.h

```diff
@@ -344,6 +344,9 @@
 
 #endif
 
+#if defined(USE_TREMOR) && !defined(USE_VORBIS)
```

Comment:
USE_VORBIS is already defined by configure when using Tremor, as well as by all the stand-alone Makefiles that use Tremor. As such, this should be fine.

Reply:
OK... To be clear, this means that when USE_TREMOR is defined, this code _forces_ USE_VORBIS to be defined. Correct? Just that the create_project changes seem to be trying to stop both being defined at the same time... so a bit confusing. ... plus not sure why you need to check "!defined(USE_VORBIS)" since you then force it to defined anyway....

Reply:
If this can be resolved, then am happy to merge this as enough time has passed for others to review or comment. and the rest of the changes look fine.

Reply:
> Just that the create_project changes seem to be trying to stop both being defined at the same time... so a bit confusing. ...

This is to stop both libraries from being _linked_ simultaneously, as they conflict with each other. USE_VORBIS still needs to be defined as the code that uses libvorbis/tremor only checks for USE_VORBIS, but not USE_TREMOR.

Reply:
Ah OK... Well then happy to merge.